SU-SWS/stanford_layouts
# Earth 248 fixes

# READY FOR REVIEW

# Summary
- Fixes for #1 

* Adds css compilation path for aiur to gruntfile.js
* Adds closing " to template file
* Renames the plakes-left.scss file to aiur.scss
* Removes all of the plakes css from aiur
* Adds auir library so that aiur.css gets loaded when aiur template is used.

# Needed By (Date)
- Thursday July 6th

# Urgency
- Soonish

# Steps to Test
1. Review PR
2. Merge in to a temporary branch based from EARTH-248-sidebar
3. Install grunt if you have not already (npm install)
4. Run `grunt sass` to ensure compilation and no changes to css

# Associated Issues and/or People
- Pull #1 and all other tickets associated with that.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
